scratch.R

```diff
@@ -1,9 +1,8 @@
 
-df <- create_validation_docs(
-  read_requirements_gsheet(
-    "1ScdlEACqC-ypJN1k8m9tjDmFIg1UfzMDyCKOzzOCUTg",
-    title_col = "ReqID", story_col = "Requirement",
-    risk = "Category",  # bogus for now
-    test_ids_col = "TestID"),
-  "validation-results-sample/"
+spec <- read_spec_gsheets(
```

Comment:
it would be great to have a googlesheets test in here, but that would require building in some credentials, right? Or maybe it's just "this test will only work if you have your creds set up in your session"?

Not sure that's worth it at the moment, but probably something we should add.

Related: we should have some `skip_*` helpers that can skip the (currently non-existent) googlesheets tests and the (existing but currently broken) github tests. Let's try to remember to add that in a cleanup PR a bit later.

Reply:
> I assume you didn't mean to commit this file?

It's not beautiful, but I did mean to commit it to make it easier for you to run/play with it, with the intention of dropping it later.  On the same page wrt to needing tests here.